src/app/components/Balao.tsx

```diff
@@ -0,0 +1,30 @@
+import ImageG from "@/components/ImageG";
+
+interface BalaoObjeto {
+    imagem: string,
+    alternativo: string,
+    titulo: string,
+    descricao: string
+}
+
+interface BaloesInt {
+    lista: BalaoObjeto[],
+}
+
+export default function Balao(props: BaloesInt) {
+    return (
+        <div className="mt-16 grid grid-cols-3 gap-x-5 gap-y-4"> {/*não me preocupei com responsividade mas PRECISAMOS*/}
```

Comment:
**:cloud: Sugestão**

Para os comentários que são tarefas que precisaremos fazer no futuro (como adicionar responsividade ou resolver um bug), colocar `TODO: tarefa em uma frase`

Assim, quando a gente pesquisar por "TODO" no código, conseguimos ver todos os pontos de atenção que a gente precisa ter

https://medium.com/imdoneio/the-imdone-pitch-feedback-welcome-386430accf01